site/_docs/reference.md

```diff
@@ -2779,6 +2779,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b o | RPAD(string, length[, pattern ])             | Returns a string or bytes value that consists of *string* appended to *length* with *pattern*
 | b o | RTRIM(string)                                | Returns *string* with all blanks removed from the end
 | b | SAFE_CAST(value AS type)                       | Converts *value* to *type*, returning NULL if conversion fails
+| b | SAFE_MULTIPLY(numeric1, numeric2)              | Returns *numeric1* * *numeric2*, null is returned if overflow occurs
```

Comment:
passive voice should be avoided

don't use comma where grammar requires a semicolon

I would write "Returns *numeric1* * *numeric2*, or null on overflow"

Reply:
Done